### PR TITLE
Adding Edenspiekermann to AMA list

### PR DESCRIPTION
**Pull request by nburkley**
Adding our new AMA to the list.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -58,7 +58,8 @@
 - [Chris Missal](https://github.com/chrismissal/ama) - Maker of Internet things in Austin, TX. A goofball, nerd, and OSS lover :heart:.
 - [Acho Arnold](https://github.com/najela/ama) - Full Stack Web Developer with great passion in [Social Psychology](https://en.wikipedia.org/wiki/Social_psychology).
 - [Rob Furlong](https://github.com/robbbz/ama) - Maker of cool things with the web, freelancer.
-- [Honza Bittner](https://github.com/HoBi/ama) - High school student, web developer, Sass and Polymer ❤️'r.
```
Comment (arthurvr) [inline readme.md:61]: Don't change this one ;)

Comment (nburkley) [inline readme.md:61]: Woops, snipped the full stop - will fix.

File: readme.md [hunk 2]
```diff
@@ -59,6 +59,7 @@
 - [Acho Arnold](https://github.com/najela/ama) - Full Stack Web Developer with great passion in [Social Psychology](https://en.wikipedia.org/wiki/Social_psychology).
 - [Rob Furlong](https://github.com/robbbz/ama) - Maker of cool things with the web, freelancer.
 - [Honza Bittner](https://github.com/HoBi/ama) - High school student, web developer, Sass and Polymer ❤️'r.
+- [Edenspiekermann](https://github.com/edenspiekermann/ama) - Design agency based in Berlin and around the world.
```
Comment (arthurvr) [inline readme.md:62]: > [Edenspiekermann]

Please use your full name.

Comment (nburkley) [inline readme.md:62]: @arthurvr [Edenspiekermann](http://www.edenspiekermann.com/) is the name of our design agency.

Comment (arthurvr) [inline readme.md:62]: Ah. Makes me wonder, aren't AMAs for people instead of companies? @sindresorhus?

Comment (kevva) [inline readme.md:62]: > Ah. Makes me wonder, aren't AMAs for people instead of companies?

+1, for me at least.

Comment (sindresorhus) [inline readme.md:62]: I don't see why we should limit it to people. Many of the best AMAs on Reddit are from companies. It definitely should be a separate category though.

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Pull request closed without merging**